libs/shared/l10n/src/lib/rtl-locales.json

```
@@ -0,0 +1,18 @@
+[
+    "ar",
+    "ckb",
+    "dv",
+    "he",
+    "he-IL",
+    "ks",
+    "ps",
+    "fa",
+    "syr",
+    "ur",
+    "ug",
+    "pa",
+    "pa-IN",
+    "pa-PK",
+    "yi",
+    "arc"
+  ]
```

```suggestion
[
    "ar",
    "arc",
    "ckb",
    "dv",
    "fa",
    "he",
    "he-IL",
    "ks",
    "pa-PK",
    "ps",
    "skr",
    "syr",
    "ug",
    "ur",
    "yi"
  ]
```

A couple changes:
Punjabi (pa, pa-IN) is LTR. Added Saraiki (skr)